### ISSUE TITLE
How to administer VMQ when hosted in Fargate/other containerized platforms with no SSH access?

### ISSUE THREAD

**Issue by atlantis** [vernemq/vernemq#1669]
I'm sorry if I'm missing something, but I've read your documentation and can't figure out how to administer a VMQ cluster that's running in Fargate, since to enable the HTTP management API I need to run vmq-admin on a node that's already connected to the cluster, but because I'm using Fargate, I can't connect directly to a running container and get a shell (since the VMQ nodes don't run sshd, nor should they).

So basically I just need to run `vmq-admin api-key add key=mykeyhere` but can't figure out how to do that without a shell.  I've tried spinning up a container and overriding the CMD to `start_vernemq api-key add key=mykeyhere` but so far no luck... any suggestions for me?  Thanks much for a great product!

**Comment by ioolkos** [comment]
Hm, seems we really have to include something as proposed in this PR: https://github.com/vernemq/docker-vernemq/pull/197

I understand Fargate is a managed service where you can't `docker exec` into containers but this creates problems for a whole class of applications. Can't go get log files, can't use a command line interface. Can't re-configure an application at all.
Living the devops future, aren't we.

**Comment by atlantis** [comment]
Yes! https://github.com/vernemq/docker-vernemq/pull/197 would do it. If I can get a password set then I should be able to use the HTTP management API just fine after that (I do have network access to each container - just not the ability to `ssh` or `docker exec` into a running container).  The problems you mentioned are definitely hurdles... Logging becomes necessarily centralized, which isn't a bad thing really given that PCI/SOC 2 require that anyway.  

But yes it is a pain in the neck to not have shell access.  You're better off than many systems, however, because you already have the HTTP API!  So if someday vmq-admin could abstract away the RPC/HTTP communication layer and work equally well locally via RPC or remotely over HTTP, then you'd be all set 🥇

**Comment by atlantis** [comment]
For now I'll manually apply that patch you mentioned, so feel free to close this issue or keep it open - whatever's more helpful to you.  Thanks much!

**Comment by codeadict** [comment]
I think this is why RabbitMQ separated the cli into a standalone package
